tortoise/queryset.py

```diff
@@ -293,7 +316,6 @@ class QuerySet(AwaitableQuery[MODEL]):
         "_q_objects",
         "_distinct",
         "_having",
-        "_custom_filters",
```

Comment:
I'm having a hard time to understand why `_custom_filters` is changed to `custom_filters` in `QuerySet` subclasses. I would love to know that!

Reply:
Honestly - I don't remember :) 

I remember last time I tried to fix it - I ran in some kind of problem related to the fact, that processing of some of this fields have different logic in QuerySet and it's children, so reusing it caused some issues about joins not working properly or something like that

If you have time to experiment with that - you can try to fix it and create PR